pom.xml

```diff
@@ -2042,6 +2044,7 @@ under the License.
     <module>common</module>
     <module>core</module>
     <module>client</module>
+    <module>ide/eclipse</module>
```

Comment:
it should be

```xml
<module>ide</module>
```

there, then `ide/pom.xml` should be again multimodule, only having

```xml
<module>eclipse</module>
```

for the moment; maybe there will be also others.